src/pybind/mgr/devicehealth/module.py

```diff
@@ -25,6 +25,29 @@
 MAX_SAMPLES=500
 
 
+def get_ata_wear_level(data: Dict[Any,Any]) -> Optional[float]:
```

Comment:
might want to remove the typing annotations from the function signature. otherwise we have 
```
"2021-04-11T09:19:22.106260+0000 mon.a (mon.0) 64 : cluster [WRN] Health check failed: Module 'devicehealth' has failed dependency: name 'Dict' is not defined (MGR_MODULE_DEPENDENCY)" in cluster log
```

see https://pulpito.ceph.com/kchai-2021-04-11_09:03:26-rados-wip-kefu-octopus-testing-2021-04-11-1228-distro-basic-smithi/

Reply:
thanks; done